.jenkins/profiling.groovy

```diff
@@ -65,7 +65,7 @@ def runCompileCommand(platform, project, jobName, boolean debug=false) {
               --architecture=\$gfx_name
 
             for i in $(find . -name "*.prof" -print 2>/dev/null); do
-              python3 -m flameprof ${i} > ${i}.svg
```

Comment:
Should we use $var for bash variables to distinguish between the groovy variables? Are braces required when dereferencing groovy variables?

Reply:
Moved to separate ticket, removing. But for the future, yes, I think we should implement the pattern you mentioned.